### ISSUE TITLE
Sponsors to followup

### ISSUE THREAD

**Issue by geohacker** [sotm-asia/sotm-asia-2018#13]
We have to start getting a sense of interest from sponsors. We'll use this ticket to list people we should reach out. @maning - it'll be great if you can do first round of outreach to gauge interest. Can you help? 

Organisation |  Contact | PoC
-------------|----------|------
Mapbox |  mikel@mapbox.com | @maning 
Mapillary |  ed@mapillary.com | @maning
DigitalGlobe | |
Ola | |
Uber | |
MapmyIndia | Ankeet Bhat | @geohacker
GFDRR | vdeparday@worldbank.org | @geohacker
Swiggy | | 
Dunzo | | 
Google | | 
HOT | rebecca.firth@hotosm.org | @maning 
Facebook | drishtiep@fb.com | @maning 
Apple | Andrew Wiseman andrew_wiseman@apple.com | @maning   
Karnataka State Remote Sensing Agency | jayachandran.mani@ksrsac.in | @geohacker

Anyone else have recommendations, please update the table!

**Comment by geohacker** [comment]
cc @jinalfoflia @satyaakam

**Comment by maning** [comment]
@mapconcierge anyone from Japna community who can help?

**Comment by geohacker** [comment]
I think we need to have a sense of interest in the next few days from sponsors to pick a venue option as outline in #7. This is to avoid last minute cashflow issues. Most venues need at least 50%  advance payment.

**Comment by govvin** [comment]
I think, even before we get in touch with these potential sponsors, we should have sponsorship tiers or packages ready to offer. Nepal had a different approach to this, compared to what was done in the Philippines, or even what is was done before in Indonesia.

A special package may also be made for co-naming rights for the event.

**Comment by geohacker** [comment]
@govvin Yes but we should get a sense of the landscape before thinking of venue ranges :) - possible to still email the folks above that you've marked with a one-liner?

@jinalfoflia is working on the sponsorship deck #2 and I'll have a itemised budget end of the week.

**Comment by geohacker** [comment]
Hey everyone, we're moving sponsorship and budget details over to a secure Google drive because we don't have private repos in this Github org for now. We'll continue to use Github for tasking. https://drive.google.com/drive/u/1/folders/1TLNimMnuWDo9sbi87EMODByesDY99_0j send us a request if you'd like access - we're very liberal.

**Comment by govvin** [comment]
Are we using any template letter for the sponsorship letter?
